kernel/banner.c

```diff
@@ -30,6 +30,10 @@ void boot_banner(void)
 #endif /* defined(CONFIG_BOOT_DELAY) && (CONFIG_BOOT_DELAY > 0) */
 
 #if CONFIG_BOOT_BANNER
+#if CONFIG_BOOT_BANNER_OVERRIDE
+	printk("*** " CONFIG_BOOT_BANNER_CUSTOM " ***\n");
```

Comment:
Would have `BANNER_VERSION BANNER_POSTFIX` appended on the end too

Reply:
`BANNER_VERSION` might confuse the user, so, just adding `BANNER_POSTFIX`?

Reply:
`BANNER_VERSION` is a requirement to me, if I'm using ncs, I want to know what version of ncs, if I've created an application and set an application version, I want this to show the version of that application

Reply:
> BANNER_VERSION is a requirement to me, if I'm using ncs, I want to know what version of ncs

FYI, in NCS it still shows Zephyr's version, but that discussion is for another forum.

Reply:
It does... *for now* :) that can be adjusted